routes/movieRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const movieController = require('../controllers/movieController');
+const router = express.Router();
+
+router.get('/main', movieController.getMain);
```

Comment:
```suggestion
router.get('', movieController.getMovies);
```
페이지 단위로 개발하지 말아주세요! movie 리스트 데이터를 전달할 뿐 main 페이지를 그려주는 것이 아닙니다.

전달한 데이터가 main에서도 사용될 수 있고 다른 곳에서도 사용될 수 있어야 합니다. main이라고 못박아버릴 필요가 없을뿐더러 실제로 main 페이지 전체를 그리는데에 필요한 데이터를 전부 주고 있지도 않습니다.

Reply:
네 코멘트 감사합니다. 반영하겠습니다.